### PR TITLE
FEATURE: add google analytics tracking per-site (to allow neoscon.io tracking); and allow google tag manager

### PR DESCRIPTION
**Pull request by skurfuerst**
... we should maybe add a cookie consent as well!

### CODE REVIEW

File: DistributionPackages/Neos.NeosIo/Resources/Private/Templates/Page/Default.html [hunk 1]
```diff
@@ -37,26 +57,27 @@
...
-					m = s.getElementsByTagName(o)[0];
-				a.async = 1;
-				a.src = g;
-				m.parentNode.insertBefore(a, m)
-			})(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
-			]]>
-
-			ga('create', '{googleAnalyticsTrackingCode}', 'auto');
-			ga('send', 'pageview');
+	<f:if condition="{neos:rendering.live()}">
```
Comment (Sebobo) [inline DistributionPackages/Neos.NeosIo/Resources/Private/Templates/Page/Default.html]: You don't need the tracking code anymore if you have the TagManager, as you configure the tracking there.

**Comment by Sebobo** [comment]
I think we should then completely remove everything related to the tracking code and just use the TagManager. Or future selfs of us will be confused again.

**Comment by skurfuerst** [comment]
@Sebobo ah OK; I did not know that they "supersede" each other or so :D agreed then!

**Comment by kdambekalns** [comment]
So… this can be closed?

**Comment by Sebobo** [comment]
I'll check what's needed and either modify or create a new PR

**Comment by Sebobo** [comment]
I'll add tag manager with a separate PR, #252 fixed the tracking for all 3 domains.

**Pull request closed without merging**